Plugins/Animation/CMakeLists.txt

```diff
@@ -70,3 +70,8 @@ target_link_libraries( Animation
                      )
 
 
+add_custom_target( Anim_data_dirs
+    COMMAND ${CMAKE_COMMAND} -E make_directory "${RADIUM_BUNDLE_DIRECTORY}/${CMAKE_BUILD_TYPE}/bin/animFrames"
```

Comment:
The value
```cmake
${RADIUM_BUNDLE_DIRECTORY}/${CMAKE_BUILD_TYPE}/bin
```
is already available as
```cmake
CMAKE_RUNTIME_OUTPUT_DIRECTORY
```
Please replace.

Reply:
well, this is set to `RADIUM_PLUGIN_OUTPUT_PATH` in each plugin's cmake... is this something to change?

Reply:
Interesting indeed.
Yes this needs to be fixed.
The variables (`CMAKE_RUNTIME_OUTPUT_DIRECTORY`,...) shouldn't be modified in each Plugin CMakeList.
If needed, directly use `RADIUM_PLUGIN_OUTPUT_PATH` and directly set the required targets properties.

Reply:
shall we do this in this PR? I remember reordering such things within the test_install* branches about the installation procedure of Radium.

Reply:
Ok, let's keep it for another PR, when we will rework the build system (install)